libs/etterlatte-trygdetid-model/src/main/kotlin/Trygdetid.kt

```diff
@@ -58,4 +59,54 @@ data class BeregnetTrygdetidGrunnlagDto(
     val dager: Int,
     val maaneder: Int,
     val aar: Int
+)
+
+data class Broek<T>(
```

Comment:
Det er alltid `Int`, sant? Gir det mening å kreve at eksterne applikasjoner må ta stilling til det også?

Reply:
Var bare at det virket litt stort antakelse at det kommer aldri noe annet form.

Hvis vi vil ha det uten generics - endre navn til IntBroek kanskje?

Reply:
Det kan du ogst, evt `trygdetidsBroek`?
Kan ikke se for meg at dette endrer seg noengang.